WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -63,10 +79,10 @@ local methods = {
             elseif(IsShiftKeyDown()) then
                 local editbox = GetCurrentKeyBoardFocus();
                 if(editbox) then
-				    if (not fullName) then
-				      local name, realm = UnitFullName("player")
-					  fullName = name.."-"..realm
-					end
+                    if (not fullName) then
```

Comment:
Can you split up whitespace clean ups into separate commits? (The formating is funky in places, and I'm okay with correcting that. But it makes it harder to review patches.)